.github/CODEOWNERS

```diff
@@ -223,8 +223,20 @@ tests/tracer/                                      @DataDog/apm-sdk-capabilities
 tests/tracer/test_ci.py                            @DataDog/ci-app-libraries
 
 # Serverless
-tests/internal/test_serverless.py   @DataDog/apm-core-python @DataDog/apm-serverless
-tests/contrib/aws_lambda            @DataDog/apm-serverless
```

Comment:
Why was this line removed?

Reply:
- `tests/internal/test_serverless.py` was moved: https://github.com/DataDog/dd-trace-py/blob/34de6cc06bf62dd8ab004c1de7ecaade48e3f5d2/.github/CODEOWNERS#L239
- `tests/contrib/aws_lambda` was changed to `tests/contrib/internal/aws_lambda` since the `tests/contrib/aws_lambda` directory no longer exists

Reply:
My mistake, I applied the `internal` subdirectory that is under `ddtrace` to `tests` as well. I updated the tests directories to be correct (`tests/contrib/*`) in addition to adding `ddtrace/contrib/internal/aws_lambda` to the list of directories with an apm-serverless codeowner.